src/vaults/SectorVault.sol

```diff
@@ -41,6 +43,9 @@ contract SectorVault is ERC4626, BatchedWithdraw {
 	address[] strategyIndex;
 	uint256 public totalStrategyHoldings;
 
+	address[] bridgeQueue;
+	uint256 balanceBeforeCrossDeposit = 0;
```

Comment:
this should actually be replaced by `availableDeposits` and tracked a little differently  (i'll work on this)